optional_plugins/runner_remote/avocado_runner_remote/__init__.py

```diff
@@ -254,51 +254,6 @@ def run(self, command, ignore_status=False, quiet=True, timeout=60):
                                                hosts=[self.hostname])
             return return_dict[self.hostname]
 
-    @staticmethod
-    def _run(command, ignore_status=False, quiet=True, timeout=60):
-        result = process.CmdResult()
-        start_time = time.time()
-        end_time = time.time() + (timeout or 0)   # Support timeout=None
-        # Fabric sometimes returns NetworkError even when timeout not reached
-        fabric_result = None
-        fabric_exception = None
-        while True:
-            try:
-                fabric_result = fabric.operations.run(command=command,
-                                                      quiet=quiet,
-                                                      warn_only=True,
-                                                      timeout=timeout,
-                                                      pty=False)
-                break
-            except fabric.network.NetworkError as details:
-                fabric_exception = details
-                if 'not found in known_hosts' in details.message:
```

Comment:
Right.  `run()` at the module top level is the other one.  But, I spotted a difference at least in this line, that is, it's not present in the top level `run()` function.  Can you double check why?